doc/en/user/source/extensions/controlflow/index.rst

```diff
@@ -48,6 +48,24 @@ A few examples::
   ows.wms.getmap=8
   # don't allow more than 2 WFS GetFeature requests with Excel output format
   ows.wfs.getfeature.application/msexcel=2
+  
+Request priority support
+........................
+
+Requests controlled by "ows.*" controllers above can be also executed in priority order, in case there are too many
+the request will block and wait, and will we awoken in priority order (highest to lowest).
+
+Currently the only way to specific a priority for a request is to add it to a request HTTP header::
+
+  ows.priority.header=<headerName>,<defaultPriority>
```

Comment:
The code is expecting **http** instead of **header**:
```
ows.priority.http=<headerName>,<defaultPriority>
```
... not sure which one is the correct one :stuck_out_tongue:

Reply:
The code, thanks for spotting this!